src/databricks/labs/ucx/source_code/directfs_access.py

```diff
@@ -146,7 +151,9 @@ def dump_all(self, dfsas: Sequence[T]):
 
     def _try_fetch(self) -> Iterable[T]:
         sql = f"SELECT * FROM {escape_sql_identifier(self.full_name)}"
-        yield from self._backend.fetch(sql)
+        for row in self._backend.fetch(sql):
+            yield self._klass.from_dict(row.as_dict())
```

Comment:
do you decode the LineageAtom in `from_dict`?

Reply:
yes